catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -90,7 +95,7 @@
 
     private String registryId = "";
 
-    private String remoteName = "";
+    private String id = "";
```

Comment:
Shouldn't need a private id string since this class inherits one from DescrbableImpl